components/Avatar/avatar.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="avatar.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="avatar">
+
+            <div class="avatar lg-avatar-size"> <img src="/images/scorpio-sign.jpg" alt="avatar"
+                    class="image-round image-responsive"> </div>
+            <div class="avatar md-avatar-size"> <img src="/images/scorpio-sign.jpg" alt="avatar"
+                    class="image-round image-responsive"> </div>
```

Comment:
img tags does not have a closing /

Reply:
I have closed the img tag with <img/> tag ,hope it will fix the issue .